packages/core/hooks/useCalendarEvents.ts

```
@@ -0,0 +1,89 @@
+// Query keys for calendar events
+export const calendarEventsKeys = {
+  all: ['calendarEvents'] as const,
+  lists: () => [...calendarEventsKeys.all, 'list'] as const,
+  list: (params?: CalendarEventQueryParams) => [...calendarEventsKeys.lists(), params] as const,
+  details: () => [...calendarEventsKeys.all, 'detail'] as const,
+  detail: (id: string) => [...calendarEventsKeys.details(), id] as const,
+};
+
+// Hook to get all calendar events
+export function useCalendarEvents(params?: CalendarEventQueryParams) {
+  return useQuery<CalendarEvent[], Error>({
```

Biome (1.9.4)

[error] Use Array<T> syntax instead of shorthand T[] syntax.

Unsafe fix: Use Array<T> syntax to replace


(lint/style/useConsistentArrayType)